web/src/app/donate/duck/index.js

```diff
@@ -0,0 +1,5 @@
+// import donateReducer from './reducers';
```

Comment:
dont commit commented out code. Only calling it out here, but applies to the commit as a whole